ckanext/ontario_theme/templates/internal/header.html

```diff
@@ -127,7 +127,7 @@
           <div class="field">
             <label for="field-sitewide-search">{% block header_site_search_label %}{{ _('Search Datasets') }}{% endblock %}</label>
             <input id="field-sitewide-search" type="text" class="form-control" name="q" placeholder="{{ _('Search') }}" />
-            <button class="btn-search" type="submit"><i class="fa fa-search"></i><span>Submit Search</span></button>
+            <button class="btn-search" type="submit" aria-label="{{ _('Submit Search') }}"><i class="fa fa-search" aria-hidden="true"></i><span class="sr-only">{{ _('Submit Search') }}</span></button>
```

Comment:
Caution on this. aria-hidden true removes this element and child elements for assistive technology. Adding a child element with sr-only implies it's intended for screen readers.